deployments/deploy-ssh.sh

```diff
@@ -1,4 +1,8 @@
 #!/bin/bash -x
+FOCUS_TESTS=VolumeReplicationIncrease,VolumeReplicationDecrease
+SCALE_FACTOR=1
+APP_LIST=elasticsearch,mysql,nginx,postgres,wordpress
+TORPEDO_IMG=thiagogonzaga/torpedo:v1.1.3
```

Comment:
Don't change any of these variables. Looks like these are from your testing script.